daml-script/export/integration-tests/reproduces-transactions/test/scala/com/daml/script/export/ReproducesTransactions.scala

```diff
@@ -307,12 +295,16 @@ trait ReproducesTransactions
       } yield ()
   }
 
-  def numParties: Int
-  def skip: Int
-  def description: String
+  "Generated export for IOU transfer compiles" - {
+    s"offset 0 - empty ACS" in { testOffset(2, 0)(testIou) }
+  }
+
+  "Generated export for IOU transfer compiles" - {
+    s"offset 4 - no trees" in { testOffset(2, 4)(testIou) }
```

Comment:
not really your fault but offset is kind of confusing as a name. I also find the whole logic here a bit weird. I'd expect that we get isolation by using different parties. Don't remember the details of this test though.

Reply:
I'd expect that we get isolation because we use different parties.
Do you think there will be a reason it is not good enough ?